core/tests/webdriverio_utils/general.js

```diff
@@ -40,7 +40,8 @@ var scrollToTop = async function() {
 };
 
 // The minimum log level we will report as an error.
-var CONSOLE_LOG_THRESHOLD = 900;
+var CONSOLE_ERROR_THRESHOLD = ['SEVERE'];
```

Comment:
Could you add a comment that points to a doc with the allowed values for this and their meanings?

Reply:
Done and btw it seems that the API mismatch came from the migration from protractor to webdriverio.